### PR TITLE
HSQLDB Integration 

### PR DESCRIPTION
**Pull request by nish-d**

### CODE REVIEW

**Review by mrigger — CHANGES_REQUESTED** [review]
Thanks, the PR looks good to me! Could you please use `mvn formatter:format` to format the source code and check whether any style violations are triggered using `mvn verify -DskipTests=True`?

**Comment by nish-d** [comment]
I have fixed all style violations and added some expression generation.

**Comment by mrigger** [comment]
Thanks! One test is currently failing: `TestUsageNamingConvention.testNonEmptyDescription`.

File: src/sqlancer/hsqldb/ast/HSQLDBUnaryPrefixOperation.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import sqlancer.Randomly;
+import sqlancer.common.ast.BinaryOperatorNode;
+import sqlancer.common.ast.newast.NewUnaryPrefixOperatorNode;
+import sqlancer.common.ast.newast.Node;
+import sqlancer.hsqldb.HSQLDBSchema;
+
+public class HSQLDBUnaryPrefixOperation extends NewUnaryPrefixOperatorNode<HSQLDBExpression> {
+
+    // private final HSQLDBUnaryPrefixOperation.HSQLDBUnaryPrefixOperator operation;
+    // private final Node<HSQLDBExpression> expression;
```
Comment (mrigger) [inline src/sqlancer/hsqldb/ast/HSQLDBUnaryPrefixOperation.java:12]: Perhaps you can delete the comments in the next PR?

**Review by mrigger — APPROVED** [review]
LGTM, thanks! Merging the PR.

File: src/sqlancer/hsqldb/ast/HSQLDBUnaryPrefixOperation.java (hunk 1 above)
Comment (nish-d) [reply src/sqlancer/hsqldb/ast/HSQLDBUnaryPrefixOperation.java:12]: Sure , will do

**Pull request merged**